mobile/src/main/res/values/strings.xml

```diff
@@ -196,15 +196,15 @@
     <string name="send_device_info_category_event_based_title">Event based</string>
     <string name="send_device_info_category_event_based_explanation">These Items are updated on corresponding events, e.g. an incoming phone call</string>
     <string name="send_device_info_category_schedule_based_title">Schedule based</string>
-    <string name="send_device_info_category_schedule_based_explanation">These Items are updated on a regular schedule\nNote: Lower values impair battery life more than higher ones and your device may decide to delay updates.</string>
+    <string name="send_device_info_category_schedule_based_explanation">These Items are updated on a regular schedule\nNote: Lower values for the schedule impair battery life more than higher ones and your device may decide to delay updates.</string>
```

Comment:
How about `Shorter intervals` instead of `Lower values for the schedule`? The latter sounds pretty German ;-)

Reply:
Thanks :)